src/pages/product-list-page/product-list-filter/patterns-filter/patterns-filter.js

```diff
@@ -23,9 +23,13 @@ const PatternsFilter = () => {
 
   const { patternsFilter } = filters;
 
-  const patterns = [
-    ...new Set(filterData.map((product) => JSON.stringify(product.pattern)))
-  ].map(JSON.parse);
+  const patterns = filterData
```

Comment:
What are you trying to do here, it is hard for me to understand

Reply:
I use it to create filter checkbox values

Reply:
fixed